### PR TITLE
[new release] atdts, atds, atdpy, atdj, atdgen, atdgen-runtime, atdgen-codec-runtime and atd (2.8.0)

### PR DESCRIPTION
**Pull request by mjambon**
Project page: <a href="https://github.com/ahrefs/atd">https://github.com/ahrefs/atd</a>

##### CHANGES:

* atdgen: use odoc syntax to disambiguate clashing names (ahrefs/atd#296)
* atdpy: propagate decorators on sum types to all constructor classes (ahrefs/atd#298)

### CODE REVIEW

File: packages/atdgen/atdgen.2.8.0/opam [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  "ygrek <ygrek@autistici.org>"
+]
+license: "MIT"
+homepage: "https://github.com/ahrefs/atd"
+bug-reports: "https://github.com/ahrefs/atd/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "atd" {>= "2.7.0"}
+  "atdgen-runtime" {>= "2.0.0"}
```
Comment (mseri) [inline packages/atdgen/atdgen.2.8.0/opam:76]:
```suggestion
  "atdgen-runtime" {>= "2.1.0"}
```
Packages installing atdgen > 2.1 and atdgen-runtime = 2.0.0 are failing with `Error: Unbound value Atdgen_runtime.Oj_run.missing_field`. We had to update the lower bound to all previous releases to fix this issue: https://github.com/ocaml/opam-repository/pull/21464/commits/d0b4d9527c354de1f25b9c1ea2a9ad00e7daa5df

Comment (mseri) [reply packages/atdgen/atdgen.2.8.0/opam:76]: Somehow this cannot be picked here since it would only appears in the lower-bound checks of revdeps

Comment (mjambon) [reply packages/atdgen/atdgen.2.8.0/opam:76]: great, thank you! I fixed it upstream.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**